examples/walkthrough/k8s201.md

```diff
@@ -1,26 +1,73 @@
+Table of Contents
+- [Labels](#labels)
+- [Replication Controllers](#replication-controllers)
+  - [Replication Controller Management](#replication-controller-management)
+- [Services](#services)
+  - [Service Management](#service-management)
+- [Health Checking](#health-checking)
+  - [Process Health Checking](#process-health-checking)
+  - [Application Health Checking](#application-health-checking)
+- [What's Next?](#whats-next)
+
+
+## Labels
+
+Having already learned about Pods and how to create them, you may be struck by an urge to create many, many pods.  Please do!  But eventually you will need a system to organize these pods into groups.  The system for achieving this in Kubernetes is Labels.  Labels are key-value pairs that are attached to each object in Kubernetes.  Label selectors can be passed along with a RESTful ```list``` request to the apiserver to retrieve a list of objects which match that label selector.
+
+To add a label, add a labels section under metadata in the pod definition:
+
+```
+  labels:
+    name: nginx
+```
+
+Full labeled nginx pod definition ([source](pod-nginx-with-label.yaml)):
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+  labels:
+    name: nginx
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    ports:
+    - name: http
+      containerPort: 80
+```
+
+Create the labeled pod:
+
+```sh
+kubectl create -f examples/walkthrough/pod-nginx-with-label.yaml
+```
+
+To list all pods that have the label `name=nginx`:
 
 ```sh
 kubectl get pods -l name=nginx
 ```
 
-Lists all pods who name label matches 'nginx'.  Labels are discussed in detail [elsewhere](http://docs.k8s.io/labels.md), but they are a core concept for two additional building blocks for Kubernetes, Replication Controllers and Services
+Labels are discussed in detail [elsewhere](http://docs.k8s.io/labels.md), but they are a core concept for two additional building blocks for Kubernetes, Replication Controllers and Services
+
 
-### Replication Controllers
+## Replication Controllers
 
-OK, now you have an awesome, multi-container, labelled pod and you want to use it to build an application, you might be tempted to just start building a whole bunch of individual pods, but if you do that, a whole host of operational concerns pop up.  For example: how will you scale the number of pods up or down and how will you ensure that all pods are homogenous?
+OK, now you know how to make awesome, multi-container, labelled pods and you want to use them to build an application, you might be tempted to just start building a whole bunch of individual pods, but if you do that, a whole host of operational concerns pop up.  For example: how will you scale the number of pods up or down and how will you ensure that all pods are homogenous?
```

Comment:
labeled is American English, labelled is British. I'll just make sure it's consistent.